### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by om-ha** [janisdd/vscode-edit-csv#69] [General]
Can we just take a moment and appreciate how great this VS Code extension is?

As I am doing advanced migration stuff and handling data with CSV, MacOS Numbers and MS Windows Office Excel failed me. This extension hasn't!

- It handles lots of data pretty well
- Advanced config of the CSV file (delimiter, header, ...)
- I know exactly that what I am typing is written in CSV format with no hidden magic 

P.S. There's no discussions section on this repo, so I opened this issue. Closing!

**Comment by janisdd** [comment]
Thanks for the nice feedback :)

I never used or tried discussions... thanks for bringing this up!
